cmake/FreecivDependencies.cmake

```diff
@@ -51,6 +51,20 @@ if(FREECIV_ENABLE_NLS)
   set(FREECIV_HAVE_LIBINTL_H TRUE)
 endif()
 
+# SDL2 for audio
+find_package(SDL2 QUIET)
+find_package(SDL2_mixer QUIET)
+if (SDL2_MIXER_LIBRARIES AND SDL2_LIBRARY)
+  set(AUDIO_SDL TRUE)
+  set(SDL2_PLAIN_INCLUDE TRUE)
```

Comment:
Better drop the `#ifdef` in `client/audio_sdl.cpp`

Reply:
updated

Reply:
Waiting for Travis to start...